src/utils/constants.ts

```diff
@@ -22,7 +22,7 @@ export const RESERVE_FORUM = 'https://forum.reserve.org/'
 export const REGISTER_BUGS =
   'https://reserve.canny.io/defi-surfaces-bug-reporting'
 export const RESERVE_STORAGE = 'https://storage.reserve.org/'
-export const RESERVE_API = 'https://api.reserve.org/'
+export const RESERVE_API = 'https://api-staging.reserve.org/'
```

Comment:
Add a TODO or add this as an environment variable, this can be merged into prod by mistake